backend/src/main/backend/Misc.txt

```diff
@@ -0,0 +1,32 @@
+Problems:
+--> [FIXED?]
+    the table class is too general. each instance of the class can create
+    tables for all the other tables.
+    For example, the Table object with name "tblMessages" can be used to 
+    create a new table in the database by a different name, "tblLikes."
+    * Need to make Table class more specific
+    * Store a list of Tables in Database.java?
+
+--> [IN PROGRESS]
+    App.java is not yet fully updated to be compatible with the new
```

Comment:
I wouldn't focus on necessarily making sure it all works exactly as it did before - I'd focus more on improving the Table abstraction and showing us how you'd instantiate the different table instances.